README.md

```diff
@@ -39,13 +39,7 @@ Returns undefined if the function has not been captured.
 
```

Comment:
```
__Note: this module uses a WeakMap shim, and throws on non-extensible functions on platforms that
don't have a native WeakMap implementation__
```
Needs to be removed since that is no longer correct.